src/main/java/com/ericsson/ei/frontend/WebController.java

```diff
@@ -42,6 +42,9 @@
     @Value("${ei.eiffelDocumentationUrls}")
     private String eiffelDocumentationUrls;
 
+    @Value("${ei.useSecureHttpForFrontend}")
+    private boolean httpsForFrontend;
```

Comment:
Why not name it with same name? e.g "useSecureHttpFrontend"

Reply:
I separated https for frontend and backend because they use the same properties for this configuration and it's wrong assuming.

Reply:
yes, but I mean you cna use same name in prpoperties and code.
@Value("${ei.useSecureHttpFrontend}")
private boolean useSecureHttpFrontend;

And for backend:
@Value("${ei.useSecureHttpBackend}")
private boolean useSecureHttpBackend;

Reply:
renamed